src/api.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import { sendEventToDiscordSubscribers } from './client';
+import pkg from '../package.json';
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  return res.json({
+    name: pkg.name,
+    version: pkg.version
+  });
+});
+
+router.post('/event-to-subscribers', async (req, res) => {
+  const event: string = req.body.event || '';
+  const proposalId: string = req.body.proposalId || '';
+
+  try {
+    await sendEventToDiscordSubscribers(event, proposalId);
+    return res.json({ success: true });
+  } catch (e) {
+    return res.json({ error: e });
+  }
+});
```

Comment:
@zzuziak 

- We should make sure this route is used only by us. 
- snapshot-webhook sends a secret in the `authentication` header, Example:
- ![Untitled 2](https://user-images.githubusercontent.com/15967809/215856122-5446c8c6-d126-4b3f-a0a7-5362142a57f1.png)
- We can add a variable in .env and check if the header is equal or not (for development env can be empty)
- For 
```
const proposalId: string = req.body.proposalId || '';
```
In the event payload proposal id is `proposal.id`